vendors/halo2/BUILD.bazel

```diff
@@ -14,18 +14,18 @@ tachyon_rust_library(
     crate_features = FEATUERS,
     proc_macro_deps = all_crate_deps(proc_macro = True),
     deps = all_crate_deps(normal = True) + [
-        "//tachyon/rs:tachyon_rs",
         ":bn254_blake2b_writer",
         ":bn254_cxx_bridge",
         ":bn254_evals",
         ":bn254_msm",
         ":bn254_msm_gpu",
         ":bn254_poly",
+        ":bn254_rational_evals",
         ":bn254_shplonk_prover",
         ":bn254_shplonk_proving_key",
-        ":bn254_rational_evals",
         ":xor_shift_rng",
         ":xor_shift_rng_cxx_bridge",
+        "//tachyon/rs:tachyon_rs",
```

Comment:
These changes will be deleted since #258 is merged right?

Reply:
Yes!